templates/README.md

```diff
@@ -210,7 +210,26 @@ ALTER TABLE address
  ADD COLUMN city VARCHAR(30) AFTER street_name,
  ADD COLUMN province VARCHAR(30) AFTER city
 ```
+<%if eq (index .Params `billingEnabled`) "yes" %>
+## Billing example
+A subscription and checkout example using [Stripe](https://stripe.com), coupled with the frontend repository to provide an end-to-end checkout example for you to customize. We also setup a webhook and an endpoint in the backend to receive webhook when events occur.
 
+### Setup
+We have setup for you in the stripe platform
```

Comment:
Since this will be in their repo let's remove language like "we" and "you" and switch to something like "The following example content has been set up in Stripe:"
Same with any other documentation.